CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.10.5 (Next)
 
 * Refactored `SlackRubyBot::MVC::Controller::Base`, consolidated ivar handling, centralized object allocations and DRYed up the code - [@chuckremes](https://github.com/chuckremes).
+* [#157](https://github.com/slack-ruby/slack-ruby-bot/pull/157): Added support for checking multiple responses at once via new `respond_with_slack_messages` matcher, and single response out of multiple responses to `response_with_slack_message` matcher - [@gcraig99](https://github.com/gcraig99) 
```

Comment:
We already had `respond_with_slack_message` (typo btw), so I would just say "Added `respond_with_slack_messages` RSpec expectation".

You need a period at the end of this for Danger to be happy ;)